app-accessibility/brltty/brltty-6.6-r2.ebuild

```diff
@@ -0,0 +1,223 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{10..13} )
+FINDLIB_USE="ocaml"
+JAVA_PKG_WANT_SOURCE="1.8"
+JAVA_PKG_WANT_TARGET="1.8"
+
+inherit findlib toolchain-funcs java-pkg-opt-2 autotools python-r1 tmpfiles
+
+DESCRIPTION="Daemon that provides access to the Linux/Unix console for a blind person"
+HOMEPAGE="https://brltty.app/"
+SRC_URI="https://brltty.app/archive/${P}.tar.xz"
+
+LICENSE="GPL-2 LGPL-2.1"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~arm64 ~ia64 ~loong ~ppc ~ppc64 ~riscv ~x86"
+IUSE="+api +beeper bluetooth doc +fm gpm iconv icu
+		java louis +midi ncurses nls ocaml +pcm policykit python
+		usb systemd +speech tcl xml X"
+REQUIRED_USE="doc? ( api )
+	java? ( api )
+	ocaml? ( api )
+	python? ( api ${PYTHON_REQUIRED_USE} )
+	tcl? ( api )"
+
+DEPEND="
+	acct-group/brltty
+	acct-user/brltty
+	dev-libs/libpcre2[pcre32]
+	bluetooth? (
+		sys-apps/dbus
+		net-wireless/bluez
+	)
+	gpm? ( >=sys-libs/gpm-1.20 )
+	iconv? ( virtual/libiconv )
+	icu? ( dev-libs/icu:= )
+	louis? ( dev-libs/liblouis:= )
+	midi? ( media-libs/alsa-lib )
+	ncurses? ( sys-libs/ncurses:0= )
+	pcm? ( media-libs/alsa-lib )
+	policykit? ( sys-auth/polkit )
+	python? (
+		${PYTHON_DEPS}
+		dev-python/setuptools[${PYTHON_USEDEP}]
+	)
+	speech? (
+		app-accessibility/espeak-ng
+		app-accessibility/flite
+		app-accessibility/speech-dispatcher
+	)
+	systemd? ( sys-apps/systemd )
+	tcl? ( >=dev-lang/tcl-8.6.13-r1:= )
+	usb? ( virtual/libusb:1 )
+	xml? ( dev-libs/expat )
+	X? (
+		app-accessibility/at-spi2-core:2
+		sys-apps/dbus
+		x11-libs/libX11
+		x11-libs/libXaw
+		x11-libs/libXfixes
+		x11-libs/libXt
+		x11-libs/libXtst
+	)"
+RDEPEND="${DEPEND}
+	java? ( >=virtual/jre-1.8:* )
+"
+BDEPEND="
+	>=dev-lang/tcl-8.6.13-r1
+	virtual/pkgconfig
+	java? ( >=virtual/jdk-1.8:* )
```

Comment:
`jdk` should not be here but go into DEPEND, see https://wiki.gentoo.org/wiki/Java_Developer_Guide#JDK.2FJRE

Reply:
Done in https://github.com/gentoo/gentoo/pull/34968/commits/620a5e3442c2d531bbdd07754ce02c0e9b3f58b5